### ISSUE TITLE
[bug]PageView::pageTurningEvent doesn't work

### ISSUE THREAD

**Issue by feijing566** [cocos2d/cocos2d-x#15523]
[bug]PageView::pageTurningEvent doesn't work
@zilongshanren 

The lua test project about ccui.PageViewEventType.turning is invalid

```
    local function pageViewEvent(sender, eventType)
        if eventType == ccui.PageViewEventType.turning then
            local pageInfo = string.format("page %d " , pageView:getCurPageIndex() + 1)
            self._displayValueLabel:setString(pageInfo)
        end
    end
    pageView:addEventListener(pageViewEvent)
```

the ver is v3.10

**Comment by zilongshanren** [comment]
@feijing566 
It should be fixed in v3.11...

**Comment by feijing566** [comment]
@zilongshanren 
3Q,i will test it
